tracing-appender/Cargo.toml

```diff
@@ -20,10 +20,15 @@ keywords = ["logging", "tracing", "file-appender", "non-blocking-writer"]
 edition = "2018"
 
 [dependencies]
-tracing-subscriber = {path = "../tracing-subscriber", version = "0.2.7"}
 crossbeam-channel = "0.4.2"
 chrono = "0.4.11"
 
+[dependencies.tracing-subscriber]
```

Comment:
personal opinion: I hate this style I prefer the `tracing-subscriber = { ... }`.

Reply:
It’s not the best, but the alternative was comically long.

Reply:
IMO, this style is better, because TOML doesn't allow newlines in the other form. The inconsistency is a bummer though. :woman_shrugging: